.travis.yml.disabled

```diff
@@ -17,11 +17,12 @@ env:
     - PRODUCT_TEST_GROUP=5
     - PRODUCT_TEST_GROUP=6
     - PRODUCT_TEST_GROUP=7
-install: 
+before_script:
   - pip install --upgrade pip==6.1.1
```

Comment:
not related to this change, but the pip version in requirements.txt is 7.1.2  (and maybe should be bumped up to the latest version of pip)